packages/sdk/src/doc/IGetResourceUriOptions.ts

```diff
@@ -39,4 +41,9 @@ export interface IGetResourceUriOptions {
     *    "CSDGROUP(D*)"
     */
   parameter?: string;
+
+  /**
+   *  Query parameters to be used in the HTTP request
+   */
+  queryParams?: IResourceQueryParams;
```

Comment:
I like the idea of a query parameters object.
Maybe by V4 we could move `criteria` and `parameter` to that object 😋